### DISCUSSION TITLE
Support more provisioning engines

### DISCUSSION THREAD

**Discussion by ShaiBer** [winglang/wing#6533] [General]
### Feature Spec

Please comment here if you'd like to have a provisioning engine added as a target for Wing, or add 👍 to a comment by someone else

### Use Cases

So we know how much demand there is for different provisioning engines

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by ShaiBer** [comment]
Kubernetes

[a comment by an automated account is not shown]

**Comment by mikaelvesavuori** [comment]
CloudFormation

**Comment by Chriscbr** [comment]
@mikaelvesavuori Thanks for the request! You might want to check out Wing's support for AWS CDK as a compilation target -- using this mode will produce a deployable CloudFormation template. You can find documentation about compiling Wing programs to AWS CDK [here](https://www.winglang.io/docs/tools/cli#awscdk-target).

**Comment by itssubhodiproy** [comment]
Pulumi

**Comment by selmison** [comment]
Azure Bicep

**Comment by selmison** [comment]
[Azure SDK](https://github.com/Azure/azure-sdk-for-js)

[a comment by an automated account is not shown]
